cmd/query/app/http_handler.go

```diff
@@ -80,6 +80,84 @@ type APIHandler struct {
 	tracer              trace.TracerProvider
 }
 
+type TQualityMetrics struct {
```

Comment:
please move all new code (data types and sample data) to another package like `app/qualitymetrics` and only leave the handler function in this pkg